staging/src/k8s.io/client-go/util/connrotation/connrotation_test.go

```diff
@@ -50,6 +52,73 @@ func TestCloseAll(t *testing.T) {
 	}
 }
 
+func TestCloseAllRace(t *testing.T) {
```

Comment:
Add a comment explaining what this tests, it's not immediately obvious without the issue context